### ISSUE TITLE
On macOS, Big Sur with M1 chip syntax highliting doesn't work

### ISSUE THREAD

**Issue by bmaxa** [jmacdonald/amp#231]
Compiled from source and install with brew, same. No syntax highlighting.
Also when installing from brew and putting sublime-syntax files in syntaxes
program crashes with: thread 'main' panicked at 'attempted to leave type `linked_hash_map::Node<yaml::Yaml, yaml::Yaml>` uninitialized, which is invalid', /private/tmp/rust-20210325-61749-fsgy2a/rustc-1.51.0-src/library/core/src/mem/mod.rs:671:9

**Comment by pbgc** [comment]
@bmaxa take a look at: https://github.com/jmacdonald/amp/issues/205

**Comment by bmaxa** [comment]
in  git repo this is solved, but syntax highlighting still doesn't work, or am I missing something again?

**Comment by pbgc** [comment]
@bmaxa it's not fixed in the git repo. It was not merged. 
Just switch the compiler to stable-1.47 (can be done via running rustup override set 1.47 in the amp directory) and rebuild and it will work

**Comment by bmaxa** [comment]
Compiled with stable, same. Perhaps I have to change colour scheme in terminal? Any advice?

**Comment by bmaxa** [comment]
Yes, that is. When started in tmux, syntax highlighting works alright!
So it's terminal quirk.

**Comment by pbgc** [comment]
@bmaxa Yes, I had those problems with URxvt .. but not with Alacritty.
(I thought you were still having that error linked_hash_map .... that's why I said to change compiler version)

**Comment by bmaxa** [comment]
this is from Cargo.toml I pulled couple of days ago: yaml-rust = ">= 0.4.5"
so it is fixed in repo, I hadn't had crash with nightly compiler when building from source,\only with brew installed.

**Comment by pbgc** [comment]
@bmaxa you are right. The problem was fixed upstream. I reported this problem 6 months ago and could not remember the details but had the idea that the pull request from christoph-heiss had not been merged yet (and that's true .. because it was not needed).

**Comment by Kerbidiah** [comment]
not sure if this helps, but I have the same issue (at least it sounds like its the same) on an Intel Mac (also on Big Sur). It seems to work fine in Alacrity though. I installed amp through homebrew
